src/pages/Autolending/AutolendingWho.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+	<div class="row who-area" :class="{ obscure: !isEnabled }">
```

Comment:
Theres actually not much new code here. Most of these lines were what was on the settings page, which has now been moved into the modal.